boards/arm/tmo_dev_edge/board.h

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Most of the content of this file should be moved to the devicetree and should be accessed by dt-defined macros.

Reply:
Can you give me an example of this?

Reply:
You'll need to introduce (board local) devicetree bindings for the devices and describe them in DTS. See `boards/arm/arty/dts/bindings/` for an in-tree example.